plugins/filter_kubernetes/kubernetes.c

```diff
@@ -847,6 +847,11 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct flb_kube, kubelet_port),
      "kubelet port to connect with when using kubelet"
     },
+    {
+     FLB_CONFIG_MAP_INT, "kube_token_ttl", "600",
```

Comment:
I think the default should be much higher than 10 minutes? My understanding is that these tokens take at least a day to expire/refresh by default or something? Why not make it like 10 hours?

Reply:
10 mins is just fine. Reading frequently is recommended...the frequency of refresh is determined by the kubernetes apiserver.

Reply:
I used 600s as default refresh frequency, because its the default value for the existing _Kube_Token_Command_